iphone/Classes/TiUIView.m

```diff
@@ -330,7 +330,7 @@ -(UIImage*)loadImage:(id)image
 		NSLog(@"[WARN] could not find image: %@",image);
 		return nil;
 	}
-	return [[ImageLoader sharedLoader] loadImmediateStretchableImage:url withLeftCap:leftCap topCap:topCap];
+	return [TiUtils loadCappedBackgroundImage:image forProxy:[self proxy] withLeftCap:leftCap withTopCap:topCap];
```

Comment:
`withTopCap` should be `topCap`, since the `with` prefix is only used for the 1st argument, based on Obj-C coding guidelines.